src/plugins/data_table/dataTables.colReorder.js

```diff
@@ -519,8 +527,10 @@
     ColReorder.aoInstances.push(this);
 
 
+    var tableElement = $(this.s.dt.nTable);
+
     if (this.s.bResizeTableWrapper) {
-      $(this.s.dt.nTableWrapper).width($(this.s.dt.nTable).width());
+      tableElement.width(tableElement.width());
```

Comment:
Oh thanks for catching this...I _think_ that I was trying to DRY out all the `$(this.s.dt.nTable)`s and replaced one too many